deeppavlov/utils/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,340 @@
+    'wic': 'WiC'
+}
+
+RSG_TASKS = {
+    'lidirus': 'LiDiRus',
+    'rcb': 'RCB',
+    'parus': 'PARus',
+    'muserc': 'MuSeRC',
+    'terra': 'TERRa',
+    'russe': 'RUSSE',
+    'rwsd': 'RWSD',
+    'danetqa': 'DaNetQA',
+    'rucos': 'RuCoS'
+}
+
+
+def get_predictions(model, data_gen, replace_word=None, round_res=False):
+    """Gets model predictions and replaces model output with replace_word.
+    
+    Args:
+        model: The model itself.
+        data_gen: Iterator with data to be submitted.
+        replace_word: Model outputs to be replaced with 1, other outputsare replaced with 0. If None, model outputs are not replaced.
+        round_res: If True, model outputs are rounded (used in stsb).
+    """
+
+    submission = {'index': [], 'prediction': []}
+    for idx, (x, _) in enumerate(tqdm(data_gen)):
+        prediction = model.compute(x)[0]
+        if replace_word:
+            prediction = 1 if prediction in {replace_word} else 0
+        if round_res:
+            prediction = round(prediction, 3)
+        submission['index'].append(idx)
+        submission['prediction'].append(prediction)
+    return submission
+
+
+def submit_glue(config: Union[str, Path, dict], output_path: Optional[Union[str, Path]] = None) -> None:
+    """Creates submission file for the GLUE tasks.
+    Args:
+        config: Configuration of the model.
+        output_path: Path to output file. If None, file name is selected according corresponding task name.
+    """
+
+    config = parse_config(config)
+    data = read_data_by_config(config)
+    iterator = get_iterator_from_config(config, data)
+    task_name = config['dataset_reader']['name']
+
+    data_gen = iterator.gen_batches(1, data_type='test', shuffle=False)
+
+    model = build_model(config)
+
+    if task_name in {'cola'}:
+        submission = get_predictions(model, data_gen, 'acceptable')
+
+    elif task_name in {'mnli'}:
+        if config['dataset_reader']['valid'] in {'validation_matched'}:
+            task_name = 'mnli-m'
+        else:
+            task_name = 'mnli-mm'
+        submission = get_predictions(model, data_gen)
+
+    elif task_name in {'mrpc'}:
+        submission = get_predictions(model, data_gen, 'equivalent')
+
+    elif task_name in {'sst2'}:
+        submission = get_predictions(model, data_gen, 'positive')
+
+    elif task_name in {'stsb'}:
+        submission = get_predictions(model, data_gen, None, True)
+
+    elif task_name in {'wnli'}:
+        submission = get_predictions(model, data_gen, 'entailment')
+
+    elif task_name in GLUE_TASKS:
+        submission = get_predictions(model, data_gen)
+    else:
+        raise ValueError(f'Unexpected GLUE task name: {task_name}')
+
+    save_path = output_path if output_path is not None else f'{GLUE_TASKS[task_name]}.tsv'
+    save_path = expand_path(save_path)
+    save_path.parent.mkdir(parents=True, exist_ok=True)
+    df = pd.DataFrame(submission)
+    df.to_csv(save_path, sep="\t", index=False)
```

Comment:
Replace `pandas` either with `csv` module or with `numpy`: we are going to remove pandas from library.

Reply:
Fixed in [commit](https://github.com/deeppavlov/DeepPavlov/pull/1647/commits/255b5c3fc5ef19d39b7ba6fb7f977402b9dc70a2)